defaults/main.yml

```diff
@@ -13,3 +13,9 @@ RABBITMQ_KEY_FILE: "/etc/letsencrypt/live/{{ RABBITMQ_HOSTNAME }}/privkey.pem"
 RABBITMQ_APT_PACKAGES:
   - python-software-properties
   - python-letsencrypt
+
+RABBITMQ_EXPORT_DIR: /usr/local/lib/rabbitmq/backup
+RABBITMQ_SCRIPTS_DIR: /usr/local/bin
```

Comment:
This should be `/usr/local/sbin`, since you set the permissions to only let root execute it.